src/pages/contact.tsx

```diff
@@ -2,16 +2,86 @@ import React from 'react';
 
 import Layout from '../components/layout/layout';
 import SEO from '../components/seo';
-import { PageTitle } from '../components/typography/typography';
+import { SubTitle, Text, TextLink } from '../components/typography/typography';
 import { Grid, GridItem } from '../components/grid/grid';
+import { ContactForm } from '../components/form/contact';
+import styled from 'styled-components';
+import { up } from '../components/breakpoint/breakpoint';
+import { graphql, useStaticQuery } from 'gatsby';
+import { FluidObject } from 'gatsby-image';
+
+const CONTACT_PAGE_QUERY = graphql`
+  query {
+    imagePlaceholder: file(relativePath: { regex: "/placeholder/" }) {
+      childImageSharp {
+        fluid {
+          ...GatsbyImageSharpFluid
+        }
+      }
+    }
+  }
+`;
+
+const ContactSubTitle = styled(SubTitle)<{ short?: boolean }>`
+  margin-top: ${(props) => (props.short ? '40px' : '80px')};
+
+  ${up('md')} {
+    margin-top: ${(props) => (props.short ? '80px' : '160px')};
+  }
+`;
 
 const ContactPage: React.FC = () => {
+  const data = useStaticQuery<{
+    imagePlaceholder: { childImageSharp: { fluid: FluidObject } };
+  }>(CONTACT_PAGE_QUERY);
+
   return (
-    <Layout>
+    <Layout heroImage={data.imagePlaceholder.childImageSharp.fluid}>
       <SEO title="Contact" />
-      <Grid>
-        <GridItem>
-          <PageTitle>Contact</PageTitle>
+      <Grid center>
+        <GridItem xs={0} md={2} />
+        <GridItem xs={12} md={8}>
+          <ContactSubTitle short>Adresse</ContactSubTitle>
+          <div>
```

Comment:
For my clarification. This will be replaced by markdown in the future and mapped to the very same components you use here. Is this correct?

Reply:
Yes, you are totally right.